### DISCUSSION TITLE
No targets in Prometheus UI

### DISCUSSION THREAD

**Discussion by Flaco0898** [community/community#81207] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

I really am new with prometheus so i don't really know much about it, I already have 20 servers monitoring on the prometheus.yml, but once i add one more to the list, the UI just shows me one of them, and all the Grafana metrics goes to NO DATA, just by adding one more, I thought it was a scrape time interval problem so i extended from 15s to 120s but the results are the same.
![image](https://github.com/community/community/assets/154467445/57fd8e4a-e0fa-4ea3-9506-ce3a61f52ea7)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Flaco0898 ! 

We're happy you're here. The Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. 

If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
